js/vm/typeaheadlist.js

```diff
@@ -26,20 +26,11 @@ export default class LectureList {
     }
   };
 
-  filtered() {
-    if (this.searchString === '') {
-      return this;
-    }
-    let regex = this.getSearchRegex(this.searchString);
-
-    return this.lectures.filter(l => regex.test(l.name));
-  }
-
   get typeaheadDataset() {
```

Comment:
Hm, I guess so. Of course, then we can ditch the whole `TypeaheadList` class and put the `getJSON` calls in the `DocumentSelection` constructor, which sounds even better.